docs/release-notes-pxc-v8.0.30.upd.md

```diff
@@ -0,0 +1,11 @@
+# Percona Distribution for MySQL 8.0.30 using Percona XtraDB Cluster Update (2023-01-23)
+
+| Release date    | January 23, 2023 |
+| :-------------- | :--------------- |
+|**Installation** | [Installing Percona Distribution for MySQL](installing.md)|
+
+Percona Distribution for MySQL is a single solution with the most critical enterprise components from the MySQL open source community, designed and tested to work together.
+
+Percona Distribution for MySQL provides two deployment variants: one is based on *Percona Server for MySQL* and another one is based on *Percona XtraDB Cluster*. 
+
+This update to the release of Percona Distribution for MySQL using the *Percona XtraDB Cluster* includes the new version of [Percona Toolkit 3.5.1](https://docs.percona.com/percona-toolkit/release_notes.html#v3-5-1-released-2023-01-13).
```

Comment:
same issue with the date in the URL. It should be 2023-01-23

Reply:
Updated